packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -1883,6 +1862,35 @@ export class GLTFLoader implements IGLTFLoader {
         return bufferView._data;
     }
 
+    public _loadBoundingInfo(positionAccessor: IAccessor): Nullable<BoundingInfo> {
+        if (positionAccessor.min && positionAccessor.max) {
+            const min = TmpVectors.Vector3[0].copyFromFloats(...(positionAccessor.min as [number, number, number]));
+            const max = TmpVectors.Vector3[1].copyFromFloats(...(positionAccessor.max as [number, number, number]));
+            if (positionAccessor.normalized && positionAccessor.componentType !== AccessorComponentType.FLOAT) {
+                let divider = 1;
+                switch (positionAccessor.componentType) {
+                    case AccessorComponentType.BYTE:
+                        divider = 127.0;
+                        break;
+                    case AccessorComponentType.UNSIGNED_BYTE:
+                        divider = 255.0;
+                        break;
+                    case AccessorComponentType.SHORT:
+                        divider = 32767.0;
+                        break;
+                    case AccessorComponentType.UNSIGNED_SHORT:
+                        divider = 65535.0;
+                        break;
+                }
+                const oneOverDivider = 1 / divider;
+                min.scaleInPlace(oneOverDivider);
+                max.scaleInPlace(oneOverDivider);
+            }
+            return new BoundingInfo(min, max);
+        }
+        return null;
+    }

```

Comment:
This function doesn't need to be a member function since it doesn't access members. The new guidance is to use export functions at the module level. Can you move this to the top of the file as an export function and use it from the loader extension? We are also using PascalCase (UpperCamelCase) for export functions.

Reply:
thanks for the suggestion, code is updated, please review it again.